app/page.tsx

```
@@ -14,6 +14,7 @@ import { Button } from "@/components/ui/button"
 import { Input } from "@/components/ui/input"
 import { Avatar, AvatarFallback } from "@/components/ui/avatar"
 import { useUser, useClerk, useSession } from "@clerk/nextjs"
+import { BYPASS_CLERK, DEV_USER_EMAIL } from "@/lib/dev-auth"
```

ESLint

[error] 'DEV_USER_EMAIL' is defined but never used.

(@typescript-eslint/no-unused-vars)